front/src/components/organisms/PongGame/PongGame.tsx

```diff
@@ -55,9 +66,16 @@ export default function PongGame(): React.ReactElement {
     };
   }, [userSocket]);
 
+  const leaveGame = () => {
+    history.go(-1);
```

Comment:
```suggestion
    history.goBack();
```
go(-1) === goBack() 같은 함수입니다. 더욱 가독성있게 작성하는게 좋을 것 같습니다.